src/main/java/co/omise/models/schedules/Occurrence.java

```diff
@@ -19,9 +19,9 @@
  */
 public class Occurrence extends Model {
     private String schedule;
-    @JsonProperty("schedule_date")
+    @JsonProperty("scheduled_on")
     private DateTime scheduleDate;
```

Comment:
I believe that the type of `on` properties should not be DateTime. I'm not sure what it should be, maybe `LocalDate` from Joda Time.

Reply:
Hmmm true, these have already been here and working. I'll look into what we can replace them with 👍

Reply:
OK, so I should pass on this for this PR?

Reply:
No, I'll try and make it work

Reply:
@pitiphong-p Will ping you once it's done to check it again 👌

Reply:
Used `LocalDate` and it worked fine. Will push it soon